<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_manage_window</class>
 <widget class="QDialog" name="user_manage_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserManage</string>
  </property>
  <widget class="QPushButton" name="add_b">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_b">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>del</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_l">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_l">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="admin_c">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>USR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PWD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
